### DISCUSSION TITLE
tropospheric delay correction with gacos

### DISCUSSION THREAD

**Discussion by handsomemin1201** [insarlab/MintPy#877] [Q&A]
Hi, I've been trying to use MintPY and gacos data for tropospheric delay correction recently, but it seems that I can't use the downloaded ztd file for MintPY processing directly, which means I need to do some processing of the downloaded data before using it? This question may be a little simple, but I would like to be answered, thanks.

**Comment by ditafaith** [comment]
Hi @handsomemin1201 
Have you solved the problem ?
I got an error when I use gacos files as input for atm correction.
